frontend/src/metabase/ui/components/inputs/Combobox/Combobox.module.css

```diff
@@ -0,0 +1,31 @@
+.options {
+  padding: 0.75rem;
+}
+
+.option {
+  font-size: 0.875rem;
+  padding: 0.75rem 0.5rem;
+
+  &:hover {
+    color: var(--mb-color-brand);
+    background-color: var(--mb-color-background-hover);
+  }
+}
+
+.empty {
+  color: var(--mb-color-text-light);
+  font-size: 0.875rem;
+  padding: 1.5rem 0.5rem;
+}
+
+.chevron {
+  color: inherit;
+  width: 1rem !important;
+  height: 1rem !important;
+
+  path {
+    d: path(
```

Comment:
Can we use an icon from the design system here, or at least reuse the svg path?

Reply:
This is not new styles